src/dotnet/UI.Blazor.App/Components/SubHeader/SubHeader.razor

```diff
@@ -0,0 +1,68 @@
+@using ActualChat.UI.Blazor.App.Module
+@namespace ActualChat.UI.Blazor.App.Components
+@inherits ComponentBase<AppUIHub>
+
+@if (_render) {
```

Comment:
Looks like it possible that we might the following lifecycle:
_render: false, true, false, true (again)
Can component handle this case properly?
I guess might be issues if _render is turned true due to setting IsVisible true again while OnAnimationEnd is not yet invoked. In that case we will have new sub-header rendered while js still refers to old one.